bigtable/quickstart/README.md

```diff
@@ -0,0 +1,78 @@
+# Cloud Bigtable Quickstart in Go
+
+This is a simple application that demonstrates using the [Google Cloud APIs Go
+Client Library](https://github.com/GoogleCloudPlatform/google-cloud-go) to connect
+to a Cloud Bigtable instance and read a row from a table.
```

Comment:
Maybe link to https://cloud.google.com/bigtable/?

Reply:
done